samples/analyzer/knuckles.config.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @type {import('@knuckles/config').Config}
+ */
+export default {
+  analyzer: {
+    mode: "strict",
```

Comment:
Should the sample be strict? "loose" is the default. Maybe add a new sample if needed